doc/nrf/releases_and_maturity/developing/managing_code.rst

```diff
@@ -37,6 +37,7 @@ This can be ``main`` if you want the latest state, or any released version (for
 If you omit the ``--mr`` parameter, west defaults to ``main``.
 
 This is the procedure used for :ref:`getting the nRF Connect SDK code <cloning_the_repositories>` when :ref:`manual_installation`.
+When you :ref:`install automatically <auto_installation>`, this is done by the Toolchain Manager in nRF Connect for Desktop when you :ref:`install the |NCS| <auto_installation_ncs_tcm>`.
```

Comment:
```suggestion
When :ref:`installing the |NCS| <auto_installation_ncs_tcm>` using the automatic installation method, this is handled by the Toolchain Manager in nRF Connect for Desktop.
```
An alternative phrasing option since it was a bit weird to see "when you install.." twice in one sentence.

Also |NCS| was showing up literally just as "|NCS| ", but I'm not sure why since I believe that syntax is written correctly.